tests/core/creators/manual/detached.md

```diff
@@ -0,0 +1,9 @@
+@bender-tags: 4.16.0, bug, 4461
+@bender-ui: collapsed
+@bender-ckeditor-plugins: toolbar, wysiwygarea, sourcearea, undo, clipboard, basicstyles, elementspath
+
+1. Use toggle button to detach and attach editor back.
+
+**Expected** Editor data is the same as at the beginning. Editor data is editable.
+
+**Unexpected** Editor data is empty. Editor data is not editable.
```

Comment:
I guess it will fail due to #4462 when editor is detached/attached multiple times? Maybe the scenario should be that editor is "created" on detached not and then one can attach it using button. The expected beahviour should be that it is initialized and works correctly?

Reply:
I correct the scenario description and editor creation code.